pkg/virt-handler/vm.go

```diff
@@ -1271,10 +1271,21 @@ func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.Vi
 	for _, network := range vmi.Spec.Networks {
 		networks[network.Name] = network.DeepCopy()
 	}
+	// if no networks or interfaces are found, return error
+	// this can be in the case of a new minimal VMI
+	if len(networks) == 0 || len(vmi.Spec.Domain.Devices.Interfaces) == 0 {
```

Comment:
Migrating without network connection is definitely allowed.

Reply:
We discussed it with Vatsal earlier on this PR. Is seemed like we don't get the default interface (pod via bridge binding) at this point, so "no ifaces" can mean either default or none. However, I found out we do set the defaults in admission webhook. So yes, we can just let 0 interfaces go here. Sorry @vatsalparekh for the confusion before.

Reply:
yep, defaults are set in the mutating webhook. So at this stage you will always see the final configuration.

Reply:
Removed this check now